programs/ssl/ssl_client2.c

```diff
@@ -38,6 +38,8 @@
 #define mbedtls_calloc     calloc
 #define mbedtls_free       free
 #define mbedtls_exit            exit
+#define mbedtls_calloc      calloc
+#define mbedtls_free        free
```

Comment:
`mbedtls_calloc` and `mbedtls_free` are already defined in this file. Why define them again? (Feel free to fix in PR 7/7)